autoload/limelight.vim

```diff
@@ -139,7 +139,7 @@ function! s:dim(coeff)
   let fg = synIDattr(synid, 'fg#')
   let bg = synIDattr(synid, 'bg#')
 
-  if has('gui_running') || exists('+termguicolors') || has('nvim') && $NVIM_TUI_ENABLE_TRUE_COLOR
+  if has('gui_running') || (exists('+termguicolors') && termguicolors)|| has('nvim') && $NVIM_TUI_ENABLE_TRUE_COLOR
```

Comment:
A minor nit: there's no space between `)` and `||`. And actually we don't need parentheses as `&&` takes precedence over `||`. Another question: why change `has('termguicolors')` to `exists('+termguicolors')` when the former is perfectly fine?